external/textprogressbar/textprogressbar.m

```diff
@@ -17,16 +17,18 @@ function textprogressbar(c)
 %
 %      - renamed strCR to WAITBAR_HANDLE
 %      - changed persistent type of strCR to global type
+%      - Changes To the progressbar text to be consistent (to allow
+%        intermittent output)
 
 %% Initialization
 global WAITBAR_HANDLE;           %   Carriage return pesistent variable
-
+persistent WAITBAR_PROGRESS;
```

Comment:
WAITBAR_PROGRESS, doesn't necessarily need to be set persistent, but either this, or the WAITBAR_STRING needs to be persistent, to be able to recover the old status of the progress bar, when removing/adding text.
I initially intended to recalculate the Str based on waitbar progress, but noticed, that just using the string is actually more efficient (and over that forgot to revert the WAITBAR_PROGRESS variable).